content/docs/howtos/change-background.md

```diff
@@ -0,0 +1,63 @@
+---
+title: 'Change background'
+description: >
+    Change the background
+---
+
+# Wallpaper
+
+Most Regolith Looks provide a default desktop color or image. The wallpaper can be set by specifying the path to the wallpaper image via the Xresources key `regolith.wallpaper.file`.
```

Comment:
completeness: wallpaper can also be configured by the settings app `regolith-control-center`.  Can you mention that as well for users that prefer a GUI?

Reply:
I have added a mention to the Regolith control center is this what you meant?